pkg/apis/core/v1alpha1/types_federatedtypeconfig.go

```diff
@@ -53,9 +53,9 @@ type FederatedTypeConfigSpec struct {
 	// The API resource type to be federated.
 	SourceType APIResource `json:"sourceType"`
 
-	// Configuration for StatusAggregation. If left empty, the StatusAggregation feature will be disabled.
-	// +optional
-	StatusAggregation *StatusAggregationConfig `json:"statusAggregation,omitempty"`
+	// Configuration for StatusAggregation.
+	// +required
+	StatusAggregation StatusAggregationConfig `json:"statusAggregation"`
```

Comment:
StatusAggregation is default true, unless we turn it off explicitly. To make API more clearer, I prefer to make this change.